amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/NoOpTaskExecutionListener.java

```diff
@@ -0,0 +1,22 @@
+package software.amazon.kinesis.lifecycle;
+
+import software.amazon.kinesis.leases.ShardInfo;
+
+public class NoOpTaskExecutionListener implements TaskExecutionListener {
+    /**
+     * Empty constructor for NoOp Shard Prioritization.
+     */
+    public NoOpTaskExecutionListener() {
```

Comment:
No need to have the empty constructor. Could you remove the empty lines between the brackets.

Reply:
done